kernel/kernel-impl/src/main/java/org/sakaiproject/email/impl/BasicEmailService.java

```diff
@@ -135,6 +135,9 @@
 	/** Whether to turn on mail debugging */
 	public static final String MAIL_DEBUG = "mail.debug";
 
+	public static final String MAIL_OURDOMAIN = "mail.ourdomain";
```

Comment:
> On Dec 10, 2015, at 4:06 PM, Sam Ottenhoff <notifications@github.com> wrote:
> 
> In kernel/kernel-impl/src/main/java/org/sakaiproject/email/impl/BasicEmailService.java <https://github.com/sakaiproject/sakai/pull/1438#discussion_r47285572>:
> 
> > @@ -135,6 +135,9 @@
> >  	/** Whether to turn on mail debugging */
> >  	public static final String MAIL_DEBUG = "mail.debug";
> >  
> > +	public static final String MAIL_OURDOMAIN = "mail.ourdomain";
> Can we use "serverName" as the default value here
> 
> 
I doubt it. Thinking of Rutgers. serve name will be sakai.rutgers.edu. But email addresses will generally be @rutgers.edu or @scarletmail.rutgers.edu. Checking for sakai.rutgers.edu wouldn’t be very useful.

It would be possible to have a default for the replacement address. Maybe postmaster, although I always worry that some system might consider postmaster to be low priority or spam.

> —
> Reply to this email directly or view it on GitHub <https://github.com/sakaiproject/sakai/pull/1438/files#r47285572>.
>